docs/user/reference/yaml/metrics.md

```diff
@@ -0,0 +1,248 @@
+# Metrics YAML Registry Format
+
+Metrics sent by an application or library are defined in YAML files which follow
+the [`metrics.yaml` JSON schema](https://mozilla.github.io/glean_parser/metrics-yaml.html).
+
+This files must be parsed by [`glean_parser`](https://pypi.org/project/glean-parser/) at build time
+in order to generate code in the target language (e.g. Kotlin, Swift, ...). The generated code is
+what becomes the public API to access the project's metrics.
+
+For more information on how to introduce the `glean_parser` build step for a specific language /
+environment, refer to the ["Adding Glean to your project"](../../user/adding-glean-to-your-project/index.md)
+section of this book.
+
+{{#include ../../../shared/blockquote-info.html}}
+
+## Note on the naming of these files
+
+> Although we refer to metrics definitions YAML files as `metrics.yaml` throughout Glean documentation
+> this files may be named whatever makes the most sense for each project and may even be broken down
+> into multiple files, if necessary.
+>
+> This does not apply to Firefox Desktop, where all metrics for a component must be declared in a
+> file called _exactly_ `metrics.yaml`.
```

Comment:
Nope, not anymore. They need to be listed [in the index](https://searchfox.org/mozilla-central/source/toolkit/components/glean/metrics_index.py#9-14), so any name should be fine.